cli/cli.go

```diff
@@ -533,6 +540,8 @@ Not after (expires): %s (%s)
 	AddGlobalFlag("rsh-raw", "r", "Output result of query as raw rather than an escaped JSON string or list", false, false)
 	AddGlobalFlag("rsh-server", "s", "Override scheme://server:port for an API", "", false)
 	AddGlobalFlag("rsh-header", "H", "Add custom header", []string{}, true)
+	AddGlobalFlag("rsh-requests", "R", "Do a number of requests", 1, false)
+	AddGlobalFlag("rsh-load-rate", "l", "Set a load rate in requests per second. Overrides the HTTP cache to false", 1, false)
```

Comment:
Should these default to zero? I have some concerns:

1. If non-zero by default they disable caching
2. Limiting to 1 request a second is problematic for automatic pagination (which may do many sequential requests) and also see #156 which does many requests.
3. Total number of requests vs. some time period (e.g. 30 seconds) - which is better for this?